### PR TITLE
Create a gradle "coverage" task that reports combined Unit and Integr…

### PR DESCRIPTION
**Pull request by wtritch**
…ation test coverage.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -204,8 +201,20 @@ subprojects {
...
+    }
+
+    jacocoTestReport {
+        reports {
+            xml.enabled = false
+            csv.enabled = false
+            html.enabled = true
+            html.destination = file("${reporting.baseDir}/${name}")
+        }
     }
```
Comment (wtritch) [inline build.gradle]: With this change you can still generate individual coverage reports on unit tests via:
`gradle clean test jacocoTestReport`

File: build.gradle [hunk 2]
```diff
@@ -204,8 +201,20 @@ subprojects {
...
-    tasks.withType(Test) {
-        reports.html.destination = file("${reporting.baseDir}/${name}")
+    task coverage(type: Task, dependsOn: [test, IT]) {
+        doFirst {
+            jacocoTestReport.executionData(test, IT)
+        }
+        finalizedBy(jacocoTestReport)
+    }
+
+    jacocoTestReport {
```
Comment (malaffoon) [inline build.gradle]: i know i said leave this in during the call but ... are these the same as the default settings? If so, i guess i'd get rid of it.

**Comment by chdietz** [comment]
Sounds good to me.

**Comment by agorina** [comment]
Thank you for taking care of it!

File: build.gradle (hunk 2 above)
Comment (wtritch) [reply build.gradle]: It looks like xml and csv reports are disabled by default.  It looks like the only thing we really want to configure is the html report location.  Before these changes the coverage report was found at:
`build/reports/jacoco/test/html/index.html`
After these changes the "coverage" task coverage report is found at:
`build/reports/coverage/index.html`

**Pull request merged**